src/utils/jsonValidator.js

```diff
@@ -0,0 +1,78 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  name: Joi.string().min(3).max(30).required(),
+  title: Joi.string().min(3).max(30).required(),
+  layout: Joi.string().min(3).max(30).required(),
+  caption: Joi.string().min(3).required(),
+  enabled: Joi.boolean().required(),
+  subCaption: Joi.string().min(3),
+  questions: Joi.array()
+    .required()
+    .items(
+      Joi.object({
+        name: Joi.string().required(),
```

Comment:
could we add the id that we use sometimes for the custom styles? what do you think is it worth ? 🤔

Reply:
Do you mean the general id for the form? that is already in the `schema`

Reply:
not really the id for example if you want a custom style for an input.  Like an input with a blue background and only that one you can add and id to the question like.

{
type: "input",
name: "customInput",
id: 'blueInput"
}

in styles you would attack it like: 

input {
  blueInput:{
   backgroundColor: 'blue'
   }
 }

Reply:
oh right, ok, thanks for the reminder